prow/scripts/cluster-integration/kyma-gke-load-test.sh

```diff
@@ -0,0 +1,355 @@
+            echo "ERROR: API not found, exit"
+            exit 1
+        fi
+
+        echo "Sleep for 3 seconds"
+        sleep 3
+    done
+}
+
+function generateAndExportLetsEncryptCert() {
+	shout "Generate lets encrypt certificate"
+	date
+
+    mkdir letsencrypt
+    cp /etc/credentials/sa-gke-kyma-integration/service-account.json letsencrypt
+    docker run  --name certbot \
+        --rm  \
+        -v "$(pwd)/letsencrypt:/etc/letsencrypt"    \
+        certbot/dns-google \
+        certonly \
+        -m "kyma.bot@sap.com" \
+        --agree-tos \
+        --no-eff-email \
+        --dns-google \
+        --dns-google-credentials /etc/letsencrypt/service-account.json \
+        --server https://acme-v02.api.letsencrypt.org/directory \
+        --dns-google-propagation-seconds=600 \
+        -d "*.${DOMAIN}"
+
+    TLS_CERT=$(base64 -i ./letsencrypt/live/"${DOMAIN}"/fullchain.pem | tr -d '\n')
+    export TLS_CERT
+    TLS_KEY=$(base64 -i ./letsencrypt/live/"${DOMAIN}"/privkey.pem   | tr -d '\n')
+    export TLS_KEY
+
+}
+
+function installKyma() {
+
+	kymaUnsetVar=false
+
+	for var in REMOTEENVS_IP_ADDRESS GATEWAY_IP_ADDRESS ; do
+    	if [ -z "${!var}" ] ; then
+        	echo "ERROR: $var is not set"
+        	kymaUnsetVar=true
+    	fi
+	done
+	if [ "${kymaUnsetVar}" = true ] ; then
+    	exit 1
+	fi
+
+	KYMA_INSTALLER_IMAGE="${DOCKER_PUSH_REPOSITORY}${DOCKER_PUSH_DIRECTORY}/${STANDARIZED_NAME}/${REPO_OWNER}/${REPO_NAME}:${CURRENT_TIMESTAMP}"
+
+	shout "Build Kyma-Installer Docker image"
+	date
+	KYMA_INSTALLER_IMAGE="${KYMA_INSTALLER_IMAGE}" "${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}"/create-image.sh
+
+	KYMA_RESOURCES_DIR="${KYMA_SOURCES_DIR}/installation/resources"
+	INSTALLER_YAML="${KYMA_RESOURCES_DIR}/installer.yaml"
+	INSTALLER_CONFIG="${KYMA_RESOURCES_DIR}/installer-config-cluster.yaml.tpl"
+	INSTALLER_CR="${KYMA_RESOURCES_DIR}/installer-cr-cluster.yaml.tpl"
+	
+
+	DOMAIN="${DNS_SUBDOMAIN}.${DNS_DOMAIN%?}"
+	export DOMAIN
+    generateAndExportLetsEncryptCert
+
+	shout "Apply Kyma config"
+	date
+
+	"${KYMA_SCRIPTS_DIR}"/concat-yamls.sh "${INSTALLER_YAML}" "${INSTALLER_CONFIG}" \
+		| sed -e 's;image: eu.gcr.io/kyma-project/.*/installer:.*$;'"image: ${KYMA_INSTALLER_IMAGE};" \
+		| sed -e "s/__DOMAIN__/${DOMAIN}/g" \
+		| sed -e "s/__REMOTE_ENV_IP__/${REMOTEENVS_IP_ADDRESS}/g" \
+		| sed -e "s#__TLS_CERT__#${TLS_CERT}#g" \
+		| sed -e "s#__TLS_KEY__#${TLS_KEY}#g" \
+		| sed -e "s/__EXTERNAL_PUBLIC_IP__/${GATEWAY_IP_ADDRESS}/g" \
+		| sed -e "s/__SKIP_SSL_VERIFY__/true/g" \
+		| sed -e "s/__VERSION__/0.0.1/g" \
+		| sed -e "s/__.*__//g" \
+		| kubectl apply -f-
+
+	waitUntilInstallerApiAvailable
+
+	shout "Trigger installation"
+	date
+
+    sed -e "s/__VERSION__/0.0.1/g" "${INSTALLER_CR}"  | sed -e "s/__.*__//g" | kubectl apply -f-
+	kubectl label installation/kyma-installation action=install
+	"${KYMA_SCRIPTS_DIR}"/is-installed.sh --timeout 30m
+}
+
+function waitUntilHPATestIsDone {
```

Comment:
why we have completely different approach than in testing.sh? There we just write:
```helm test application-connector --timeout 600```
which is much simpler IMO.